js/config.js

```diff
@@ -182,6 +187,159 @@ function saveUserConfig(fieldName, newValue, oldConfig) {
     });
 }
 
+/**
+ * Method to update multiple user config field
+ * @param {Object} newGlobalConfig - The latest config changes from installer
+ * @param {Object} oldUserConfig - The old user config data
+ * @returns {Promise}
+ */
+function updateUserConfig(newGlobalConfig, oldUserConfig) {
+    return new Promise((resolve, reject) => {
+        // Picking some values from global config to overwrite user config
+        const configDataToUpdate = pick(newGlobalConfig, ['url', 'minimizeOnClose', 'launchOnStartup', 'alwaysOnTop']);
+        const updatedUserConfigData = Object.assign(oldUserConfig, configDataToUpdate);
+        const jsonNewConfig = JSON.stringify(updatedUserConfigData, null, ' ');
+        // get user config path
+        let userConfigFile;
+
+        if (isMac) {
```

Comment:
why something special for user config update - saveUserConfig doesn't have to do anything special?